tests/python/gaia-ui-tests/gaiatest/apps/contacts/app.py

```diff
@@ -32,7 +32,7 @@ def launch(self):
                 *self._settings_button_locator))))
 
     def switch_to_contacts_frame(self):
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == self.name)
+        self.wait_to_be_displayed()
         self.apps.switch_to_displayed_app()
 
```

Comment:
I think we need to change in the Launch maybe http://mxr.mozilla.org/gaia/source/tests/python/gaia-ui-tests/gaiatest/tests/unit/test_launch_via_entry_point.py#13 ?